Tasks/string_builder.py

```diff
@@ -0,0 +1,52 @@
+class StringBuilder:
+
+    def __init__(self, data):
+        self.string = list(data)
+
+    def append(self, elem: str):
+        self.string.extend(list(elem))
+
+    def __str__(self):
+        return ''.join(self.string)
+
+    def insert(self, elem: str, ind: int = 0):
+        if ind > len(''.join(self.string)):
```

Comment:
```suggestion
        if ind > len(self.string):
```
why do you need to construct the string by join to check its length?